### PR TITLE
Enable Sve.CreateBreakPropagateMask() to take both operands as zero

### LINKED ISSUE #106124: JIT SVE: Assertion failed ''targetReg != embMaskOp1Reg'' during 'Generate code'
```C#

using System;
using System.Numerics;
using System.Runtime.CompilerServices;
using System.Runtime.Intrinsics;
using System.Runtime.Intrinsics.Arm;
using Xunit;

namespace JIT.HardwareIntrinsics.Arm._Sve
{
    public static partial class Program
    {
        [Fact]
        public static void TestEntryPoint()
        {
            var vr19 = Vector128.CreateScalar(0L).AsVector();
            var vr25 = Sve.CreateBreakPropagateMask(vr19, vr19);
            Consume(vr25);
        }

        [MethodImpl(MethodImplOptions.NoInlining)]
        private static void Consume(Vector<long> v)
        {
            ;
        }

    }
}
```

cc @dotnet/jit-contrib @dotnet/arm64-contrib

### PR DESCRIPTION
**Pull request by SwapnilGaikwad**
Fixes: #106124

@a74nh @kunalspathak @dotnet/arm64-contrib @arch-arm64-sve @TIHan @amanasifkhalid

### CODE REVIEW

File: src/tests/JIT/Regression/JitBlue/Runtime_106124/Runtime_106124.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
```
Comment (jakobbotsch) [inline src/tests/JIT/Regression/JitBlue/Runtime_106124/Runtime_106124.cs:4]: I assume this is a Fuzzlyn found example? Can you add the Fuzzlyn header?

Comment (SwapnilGaikwad) [reply src/tests/JIT/Regression/JitBlue/Runtime_106124/Runtime_106124.cs:4]: Well, yes and no 🙂 . I was trying to minimise [this](https://github.com/dotnet/runtime/issues/106124#issue-2455638331) example when I encountered this issue. The original example doesn't have this pattern so didn't add it in the header. Happy to change to whatever you suggest 👍

Comment (jakobbotsch) [reply src/tests/JIT/Regression/JitBlue/Runtime_106124/Runtime_106124.cs:4]: Do you mean while trying to minimize #105474? I did hit this assert during Fuzzlyn runs as well, but I assumed it was the same underlying problem, so I didn't open an issue about it.
Fine to leave it without, I just noticed the variable names looked Fuzzlyn generated, and we like to be able to track which issues are fuzzer found.

Comment (SwapnilGaikwad) [reply src/tests/JIT/Regression/JitBlue/Runtime_106124/Runtime_106124.cs:4]: Sorry, I meant - minimising #105720.

**Review by a74nh — APPROVED** [review]
LGTM.

**Review by amanasifkhalid — APPROVED** [review]
LGTM too, thanks!

File: src/tests/JIT/Regression/JitBlue/Runtime_106124/Runtime_106124.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+using Xunit;
+
+namespace JIT.HardwareIntrinsics.Arm._Sve
+{
+    public static partial class Program
+    {
+        [Fact]
+        public static void TestEntryPoint()
+        {
+            var vr19 = Vector128.CreateScalar(0L).AsVector();
```
Comment (amanasifkhalid) [inline src/tests/JIT/Regression/JitBlue/Runtime_106124/Runtime_106124.cs:18]: @SwapnilGaikwad could you please wrap this with a `Sve.IsSupported` check? Looks like this test is trying to run on unsupported hardware in CI.

File: src/tests/JIT/Regression/JitBlue/Runtime_106124/Runtime_106124.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+using System;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.Arm;
+using Xunit;
+
+namespace JIT.HardwareIntrinsics.Arm._Sve
+{
+    public static partial class Program
```
Comment (jakobbotsch) [inline src/tests/JIT/Regression/JitBlue/Runtime_106124/Runtime_106124.cs:13]: Please give this a name consistent with other regression tests, i.e. `Runtime_<Id>`. You can remove the namespace too

Comment (jakobbotsch) [reply src/tests/JIT/Regression/JitBlue/Runtime_106124/Runtime_106124.cs:13]: You can also remove `partial`

File: src/tests/JIT/Regression/JitBlue/Runtime_106124/Runtime_106124.cs (hunk 2 above)
Comment (SwapnilGaikwad) [reply src/tests/JIT/Regression/JitBlue/Runtime_106124/Runtime_106124.cs:18]: Done.

File: src/tests/JIT/Regression/JitBlue/Runtime_106124/Runtime_106124.cs (hunk 3 above)
Comment (SwapnilGaikwad) [reply src/tests/JIT/Regression/JitBlue/Runtime_106124/Runtime_106124.cs]: Done.

**Review by amanasifkhalid — APPROVED** [review]

**Pull request merged**